### PR TITLE
Fix: #99 Change jquery CSS functions in main.js

### PR DESCRIPTION
**Pull request by y-martinez**
Fix: #99

### CODE REVIEW

File: public/scripts/main.js [hunk 1]
```diff
@@ -92,10 +92,11 @@ function handle(e) {
     ' code=' + e.code
 
   if(e.code.toLowerCase()=="space"){
-    $("#space").toggleClass("pressed");
+    document.querySelector('#space').classList.toggle("pressed");
+    //document.querySelector('#space').classList.toggle('pressed',e.code.toLowerCase()=="space")
```
Comment (vegetabill) [inline public/scripts/main.js:96]: Please remove this commented line from the PR.

**Review by vegetabill — CHANGES_REQUESTED** [review]
The diff looks good but when testing with Chrome I saw some errors:

```
Uncaught TypeError: Cannot read property 'classList' of null
```

Unfortunately it's hard to tell from the jQuery which elements may or may not exist but hopefully but following the line numbers and some trial and error you can add a guard clause for any that might need one.

File: public/scripts/main.js (hunk 1 above)
Comment (y-martinez) [reply public/scripts/main.js:96]: Ohh sorry I forgot that I commented on it

**Comment by y-martinez** [comment]
> The diff looks good but when testing with Chrome I saw some errors:
> 
> ```
> Uncaught TypeError: Cannot read property 'classList' of null
> ```
> 
> Unfortunately it's hard to tell from the jQuery which elements may or may not exist but hopefully but following the line numbers and some trial and error you can add a guard clause for any that might need one.

Well, I just see the problem, it happens on some occasions, as you say it is difficult to track it but I am doing it, can I rewrite the `handle` and `release` functions?

**Comment by vegetabill** [comment]
@y-martinez if the issue is because of bad code in those functions, go ahead and rewrite them if you think that's the best way to fix the issue.

**Comment by y-martinez** [comment]
> @y-martinez if the issue is because of bad code in those functions, go ahead and rewrite them if you think that's the best way to fix the issue.

Hi @vegetabill I already refactored these functions but I didn't test it with the command key because I don't have a mac, please could you test it and give me your review. Thanks a lot.

**Comment by y-martinez** [comment]
Hi, @alodahl sorry for the inconvenience, but it would be possible that meanwhile Bill finishes reviewing the PR, you could add the label `hacktoberfest-accepted` to the PR so that the verification time of the Hacktoberfest begins because now it is two weeks of verification.

**Review by vegetabill — APPROVED** [review]
Thanks for cleaning up that function, @y-martinez . This looks good to merge, @alodahl

**Comment by vegetabill** [comment]
And please label with `hacktoberfest-accepted` (lost my labeling perms)

**Pull request merged**